books.js

```diff
@@ -0,0 +1,17 @@
+export default class Books {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(book) {
+    this.books.push(book);
+    localStorage.setItem('listBooks', JSON.stringify(this.books));
+  }
+
+  deleteItem(id) {
+    const book = document.getElementById(id);
+    book.remove();
+    this.books.splice(id, 1);
+    localStorage.setItem('listBooks', JSON.stringify(this.books));
+  }
+}
```

Comment:
- [ ] **Required:** Great job! 👏 I have noticed that this method doesn't function correctly. It cannot be relied upon to remove the correct item. Kindly see the gif below and fix the behaviour.

![delete misbahavior](https://user-images.githubusercontent.com/27709832/164110082-df848daf-5b1f-4930-8b30-bc9d540e8415.gif)